uint64 timestamp

int32 device_id

float32 x_offset
float32 x_scale
float32 y_offset
float32 y_scale
float32 z_offset
float32 z_scale
